chapters/develop-organize/guides/reading/README.md

```diff
@@ -5,31 +5,56 @@ Reviewer: Teo Duțu
 
 Guides are pieces of content consisting of detailed steps that are commonly used in practice.
 
-Guides are to be used by learners as tutorials, part of the learning process.
-They can be used as tutorial during practical session, with assistance provided to further explaining certain items, or as unassisted self-study.
+Guides are to be used by learners as **tutorials**, part of the learning process.
+They can be used as tutorials during ["Practical Sessions"](../../../use-deliver/practical-sessions/reading/README.md), either as unassisted self-study, or with assistance provided for better understanding.
 
-Guides can also by educators when demonstrating a topic or showcasing a real-world scenario to a live audience.
+Guides can also by educators as **demos**, when demonstrating a topic or showcasing a real-world scenario to a live audience.
 These demos are typically good ways to capture the attention of learners due to their real-world nature.
 
 Guides consist of:
 
-- text: created by following the guidelines in [the "Reading" section](../../reading/reading/README.md)
-- media: created by following the guidelines in [the "Media" section](../../media/reading/README.md)
-- support files: source code, skeleton files, application support files, fill-in-the blanks, templates
-
-A learner would read the text, view media files and use (hands-on) practice files to get replicate a practical result present by the guide.
+- description and walkthrough:
+  This are the rationale & goals of the guide (how it would help the learner) and the detailed set of steps that the learner must undertake to reach the goal.
+  They are created in Markdown format by following the guidelines in [the "Reading" section](../../reading/reading/README.md)
+- media:
+  These form the visual support, assisting the learner while walking through the guide.
+  They are created by following the guidelines in [the "Media" section](../../media/reading/README.md)
+- support files:
+  These form any support files that the learner or educator uses in reach the guide goal.
+  They may be source code, skeleton files, application support files, fill-in-the blanks, templates.
+
+A learner would read the description and walkthrough, view media files and use (hands-on) practice files to replicate the practical results presented by the guide.
 When used by an educator as a demo, support files are used for the live delivery;
-text and media are used by the educator to prepare before delivery.
+description, walkthrough and media are used by the educator to prepare before delivery.
```

Comment:
It seems to me that you're beating a dead horse here. It's already clear enough what teachers are supposed to do with guides and furthermore, they're free to do whatever they want anyway.